_episodes/identifiers.md

```diff
@@ -77,6 +79,8 @@ Linked data
 
 Remember that we said that data should be *fair* to computers? By assigning PIDs to our datasets (and publications), large networks of linked data and information can be created automatically.
 
+_AK: The sentences above are rather vague. If you wish to briefly mention Linked (Open) Data then why not refering to [this](https://www.w3.org/DesignIssues/LinkedData.html) document by the inventor._
+
```

Comment:
Thanks for the suggestion, could you update the text that you would like to clarify?

```suggestion
```